src/lib/inc_generated/ndpi_asn_riotgames.c.inc

```diff
@@ -0,0 +1,43 @@
+/*
+ *
+ * This file is generated automatically and part of nDPI
+ *
+ * nDPI is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * nDPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with nDPI.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/* ****************************************************** */
+
+
+static ndpi_network ndpi_protocol_riotgames_protocol_list[] = {
+ { 0x2BE54000 /* 43.229.64.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x2D072400 /* 45.7.36.0/23 */, 23, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x2D072700 /* 45.7.39.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x2DFAD000 /* 45.250.208.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x42972100 /* 66.151.33.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x67DB8000 /* 103.219.128.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x67F0E000 /* 103.240.224.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x68A08000 /* 104.160.128.0/19 */, 19, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x8A000C00 /* 138.0.12.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x976AF800 /* 151.106.248.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x976AFC00 /* 151.106.252.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ { 0x976AFE00 /* 151.106.254.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ { 0xA2F94800 /* 162.249.72.0/21 */, 21, NDPI_PROTOCOL_RIOTGAMES },
+ { 0xB6A27800 /* 182.162.120.0/21 */, 21, NDPI_PROTOCOL_RIOTGAMES },
+ { 0xB9284000 /* 185.40.64.0/22 */, 22, NDPI_PROTOCOL_RIOTGAMES },
+ { 0xC040A800 /* 192.64.168.0/21 */, 21, NDPI_PROTOCOL_RIOTGAMES },
+ { 0xC0CF0000 /* 192.207.0.0/24 */, 24, NDPI_PROTOCOL_RIOTGAMES },
+ /* End */
+ { 0x0, 0, 0 }
+};
```

Comment:
You need to include this file in ndpi_main.c and load this list...

Reply:
forgot that, thanks